tldw_Server_API/tests/Jobs/test_fair_share_integration.py

```
@@ -0,0 +1,173 @@
+"""
+from __future__ import annotations
+
+from unittest.mock import patch
+
+import pytest
+
+from tldw_Server_API.app.core.Jobs.manager import JobManager, _get_fair_share
+from tldw_Server_API.app.core.Jobs.migrations import ensure_jobs_tables
+
+
+@pytest.fixture()
```

Ruff (0.15.6)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)